tox.ini

```diff
@@ -29,7 +29,7 @@ deps =
 
 [testenv:integration-databricks-cluster]
 basepython = python3
-commands = /bin/bash -c '{envpython} -m pytest -v tests/specs/databricks-cluster.dbtspec {posargs}; ret=$?; [ $ret = 5 ] && exit 0 || exit $ret'
+commands = /bin/bash -c '{envpython} -m pytest -v --profile databricks_cluster tests/functional/adapter/test_basic.py {posargs}; ret=$?; [ $ret = 5 ] && exit 0 || exit $ret'
```

Comment:
I like the `--profile` option! I am curious if we can change the integration tests to use this too.

Reply:
I think we can, in a separate PR, or we can contribute to `dbt-spark` if they want.

Reply:
Or https://github.com/dbt-labs/dbt-core/pull/4958 will provide the testing framework even for custom integration tests?

Reply:
@ueshin Correct, our thinking is that this framework will provide the foundation for:
- basic test cases (minimum expectations for any dbt adapter)
- optional additional test cases for more advanced functionality (we'll be gradually adding, e.g. https://github.com/dbt-labs/dbt-core/pull/4958)
- your own custom integration/functional tests, which can tap into the same reusable utilities

As far as the `--profile` option in particular: It looks like this is something we added in `dbt-spark` (https://github.com/dbt-labs/dbt-spark/blob/main/tests/conftest.py), rather than the test utilities built into `dbt-core` (since the tests there only use a single Postgres profile). @gshank Do you see merit in standardizing this for all plugin maintainers, by moving `pytest_addoption` and `dbt_profile_target` into `dbt-core`? Or must they be defined in each plugin's own `conftest.py`?

Reply:
As far as the `--profile` option in particular: It looks like this is something we added in `dbt-spark` (https://github.com/dbt-labs/dbt-spark/blob/main/tests/conftest.py), rather than the test utilities built into `dbt-core` (since the tests there only use a single Postgres profile). @gshank Do you see merit in standardizing this for all plugin maintainers, by moving `pytest_addoption` and `dbt_profile_target` into `dbt-core`? Or must they be defined in each plugin's own `conftest.py`?